tests/input/examples/schema_definition-native-array-rgb.yaml

```
@@ -0,0 +1,44 @@
+id: https://example.org/arrays
+name: arrays-temperature-example
+title: Array Temperature Example
+description: |-
+  Example LinkML schema to demonstrate a 3D DataArray of temperature values with labeled axes
+license: MIT
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  wgs84: http://www.w3.org/2003/01/geo/wgs84_pos#
+  example: https://example.org/
+
+default_prefix: example
+
+imports:
+  - linkml:types
+
+classes:
+
+  RGBImage:
+    attributes:
+      rgb:
+        range: float
+        array_info:
+          # NPtyping: NDArray[Shape["* x, * y, 3 rgb"]
+          exact_dimensions: 3
+          dimensions_info:
+            x:
+            y:
+            rgb:
+              exact_cardinality: 3
+              description: r, g, b values
+              annotations:
+                names: "[red, green, blue]"
```

```suggestion
            x:
              dimension_index: 0
            y:
              dimension_index: 1
            rgb:
              dimension_index: 2
              exact_cardinality: 3
              description: r, g, b values
              annotations:
                names: "[red, green, blue]"
```

It would be good to be explicit here. This also allows some dimensions to be unnamed.